src/main/java/org/ecocean/servlet/importer/StandardImport.java

```diff
@@ -1874,50 +1984,57 @@ public Occurrence getCurrentOccurrence(Occurrence oldOcc, Row row, Shepherd mySh
 
 
 
-
-  private void initColIndexVariables(Row firstRow) {
-    colIndexMap = makeColIndexMap(firstRow);
-
-  	unusedColumns = new HashSet<String>();
+/*
+  private void initColIndexVariables(Row firstRow, Map<String,Integer> colIndexMap, Set<String> unusedColumns, List<Integer> skipCols, HashMap<String,Integer> allColsMap, TabularFeedback feedback, Sheet sheet, boolean committing, PrintWriter out) {
```

Comment:
Remove these? There was a setter above as well that seems like it will no longer be used.